pkg/custom_detectors/custom_detectors.go

```diff
@@ -82,66 +83,98 @@ func (c *customRegexWebhook) FromData(ctx context.Context, verify bool, data []b
 	// ]
 	matches := permutateMatches(regexMatches)
 
+	g := new(errgroup.Group)
+
 	// Create result object and test for verification.
+	resultsCh := make(chan detectors.Result, maxTotalMatches)
 	for _, match := range matches {
+		match := match
+		g.Go(func() error {
+			err := c.createResults(ctx, match, verify, resultsCh)
```

Comment:
nit: We can omit the err here and just return.
`return c.createResults(ctx, match, verify, resultsCh)`